cli/app.go

```diff
@@ -1144,6 +1147,45 @@ var app = &cli.App{
 							},
 							Action: RobotsPartShellAction,
 						},
+						{
+							Name:        "cp",
+							Usage:       "copy files to and from a machine part",
+							Description: `In order to use the cp command, the machine must have a valid shell type service.`,
```

Comment:
can we add more docs here like we have for https://github.com/viamrobotics/rdk/blob/main/cli/app.go#L1160-L1167? An example or two would be super helpful. Also noting that home on the robot depends on the user (usually root) that viam is running under will be useful knowledge as well

Reply:
I'd note that there is no visual progress bar as well